Gadget/ListExModule/Licenses.txt

```diff
@@ -0,0 +1,3 @@
+[Delete.png] 
+License: http://creativecommons.org/licenses/GPL/2.0/
```

Comment:
The license text should be added instead of just the URL.
https://www.gnu.org/licenses/gpl-faq.html#WhyMustIInclude